test/data/rewards-data/migration/PreferencesForVersion12WithRewardsAndAdsDisabled

```diff
@@ -13,7 +13,7 @@
   "brave": {
     "brave_ads": {
       "ads_per_hour": "2",
-      "enabled": true,
+      "enabled": false,
```

Comment:
@tmancey This particular test was still failing even after my fix. Given that it's called `WithRewardsAndAdsDisabled` I'm thinking this needs to be `false` as I've done here, but let me know if that's wrong.